backtesting/tests/test_smc_integration.py

```
@@ -0,0 +1,530 @@
+            'open': [],
+            'high': [],
+            'low': [],
+            'close': [],
+            'volume': []
+        })
+
+        # Should handle gracefully
+        try:
+            results = backtester.run_backtest(empty_df)
+            assert results is not None
+        except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)